tests/foreman/cli/test_repository.py

```diff
@@ -331,6 +378,58 @@ def test_positive_synchronize_yum_repo(self):
     @run_only_on('sat')
     @skip_if_bug_open('bugzilla', 1152237)
     @tier2
+    def test_positive_synchronize_auth_yum_repo(self):
+        """Check if secured repository can be created and synced
+
+        @Feature: HTTP Authentication Repository
+
+        @Assert: Repository is created and synced
+        """
+        for creds in valid_http_credentials(url_encoded=True):
+            for url in (FAKE_5_YUM_REPO,):
+                url_encoded = url.format(creds['login'], creds['pass'])
+                with self.subTest(url):
+                    new_repo = self._make_repository({
+                        u'content-type': u'yum',
+                        u'url': url_encoded,
+                    })
+                    # Assertion that repo is not yet synced
+                    self.assertEqual(new_repo['sync']['status'], 'Not Synced')
```

Comment:
I wanted to have this streamlined with the other tests. Possibly we might create an issue for addressing this comment for all tests across `test_repository.py`